### DISCUSSION TITLE
Implement Fontawesome Pro in Docsy

### DISCUSSION THREAD

**Discussion by nryberg** [google/docsy#1446] [CSS and other site hacks]
I'd like to ask how to implement Fontawesome Pro into my Doscy site - I've tried downloading their webfonts and using the header `script`, but neither appears to have an effect.

Also noted in this issue: https://github.com/google/docsy/issues/1283

**Comment by chalin** [comment]
Hi! Hmm, after a quick look at this, I'd say you'll need to do the following:

- Install the FA Pro as, for example, an NPM package
- In your project's `_variables_project.scss` file, override the value of `$font-awesome-font-name`
- In your project's config, add mount points so that FA Pro resources can be accessed (look at the entries for PA Free as an example).

I haven't tried this myself. I hope this helps.

**Comment by nryberg** [comment]
I was able to implement this by creating a file named **body-end.html** in the **/layouts/partials/hooks/** folder with this code:

`<script src="https://kit.fontawesome.com/aaabbbbccc.js" crossorigin="anonymous"></script>`

In FontAwesome, create a Kit, and then within the kit, you'll find a `Set-up` tab that gives you the correct code for the `script` link.
